src/janitor.c

```diff
@@ -680,6 +680,22 @@ static void do_full_maint(evutil_socket_t sock, short flags, void *arg)
 	if (cf_pause_mode == P_SUSPEND)
 		return;
 
+	/*
+	 * Creating new pools to enable `min_pool_size` enforcement even if there are no active clients.
+	 *
+	 * If clients never connect there won't be a pool to maintain the min_pool_size on, which means we have to
+	 * proactively create a pool, so that it can be maintained.
+	 *
+	 * We are doing this for databases with forced users only, to reduce the risk of creating connections in
+	 * unexpected ways, where there are many users.
```

Comment:
wrap this comment at 80 characters:
```suggestion
	 * Creating new pools to enable `min_pool_size` enforcement even if
	 * there are no active clients.
	 *
	 * If clients never connect there won't be a pool to maintain the
	 * min_pool_size on, which means we have to proactively create a pool,
	 * so that it can be maintained.
	 *
	 * We are doing this for databases with forced users only, to reduce
	 * the risk of creating connections in unexpected ways, where there are
	 * many users.
```